src/popup/components/Modals/AccountCreate.vue

```diff
@@ -40,19 +49,36 @@ export default {
   props: {
     resolve: { type: Function, required: true },
   },
-  data: () => ({
-    PlusCircleIcon,
-    loading: false,
-  }),
-  methods: {
-    async createAccount() {
-      this.loading = true;
-      await this.$store.dispatch('accounts/hdWallet/create');
-      this.loading = false;
-      this.resolve();
-    },
+  setup(props, { root }) {
+    const { deployMultisigAccount } = useMultisig({ store: root.$store });
+    const plusCircleIcon = ref(PlusCircleIcon);
+    const loading = ref(false);
+    const multisigProgress = ref<MultisigCreationSteps>(MultisigCreationStep.preparing);
+
+    async function createPlainOldAccount() {
+      loading.value = true;
+      await root.$store.dispatch('accounts/hdWallet/create');
+      loading.value = false;
+      (props.resolve as Function)();
```

Comment:
No need to type it here. When defining the prop use this code:
```
props: {
  resolve: { type: Function as PropType<() => void>, required: true },
}
```

Reply:
we can create in types:   type VoidFunction: () => void